manim/animation/composition.py

```diff
@@ -1,28 +1,31 @@
 """Tools for displaying multiple animations at once."""
+import typing
+
 import numpy as np
 
 from ..animation.animation import Animation
-from ..mobject.mobject import Group
+from ..mobject.mobject import Group, Mobject
+from ..scene.scene import Scene
 from ..utils.bezier import interpolate
 from ..utils.iterables import remove_list_redundancies
 from ..utils.rate_functions import linear
 
 __all__ = ["AnimationGroup", "Succession", "LaggedStart", "LaggedStartMap"]
 
 
-DEFAULT_LAGGED_START_LAG_RATIO = 0.05
+DEFAULT_LAGGED_START_LAG_RATIO: float = 0.05
 
 
 class AnimationGroup(Animation):
     def __init__(
         self,
-        *animations,
-        group=None,
+        *animations: Animation,
+        group: Group = None,
```

Comment:
Not entirely sure if this is meant to be a group of mobjects. And if it is, then it would probably be a VGroup, not a Group :thinking:

Reply:
I don't think this need's to vectorised. But Idk someone who knows's about it should tell.

Reply:
The problem is that VGroup doesn't inherit from Group (it inherits from VMobject). So I guess somehthing like Union[Group, VGroup]?